.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 
 node_js:
-  - 0.12
-  - 4.0
+  - 4
+  - 6
```

Comment:
Since this is a module i rather not lock it down to a specific version. The specifity should rather be in the applications.